src/Microsoft.ML.DnnImageFeaturizer.AlexNet/AlexNetExtension.cs

```diff
@@ -21,7 +21,7 @@ public static class AlexNetExtension
         /// </summary>
         public static EstimatorChain<ColumnCopyingTransformer> AlexNet(this DnnImageModelSelector dnnModelContext, IHostEnvironment env, string outputColumnName, string inputColumnName)
         {
-            return AlexNet(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(AssemblyPathHelpers.GetExecutingAssemblyLocation(), "DnnImageModels"));
+            return AlexNet(dnnModelContext, env, outputColumnName, inputColumnName, Path.Combine(Directory.GetCurrentDirectory(), "DnnImageModels"));
```

Comment:
How does this work?

Reply:
So when we add a nuget reference to any of the resnets, these resnets are automatically copied to the output folder of that project, and they can be accessed through ```Directory.GetCurrentDirectory()```

The Assembly.codebase was not working when we were trying to use it as a nuget package because it was returing the system nuget package path i.e. C:\users\.nuget\packagename\version\

Reply:
`Directory.GetCurrentDirectory()` is not going to be correct most of the time. There is nothing that says the process's current directory is the same as where the app is executing from.

Reply:
I modified it to use ```AppDomain.CurrentDomain.BaseDirectory```

Reply:
have you tried it with a .NET Core 2.1 app when consuming the NuGet package? I don't think `
`AppDomain.CurrentDomain.BaseDirectory` is going to work in that scenario. The model will be in the NuGet cache, and the BaseDirectory is where the app is running from, which aren't the same.

Reply:
I tried it with 2.1 and its working.

Reply:
Ah, I see why - I forgot that we added this part:

https://github.com/dotnet/machinelearning/blob/f78af891ea3339ec9ecabe2f35c965e8861b316d/pkg/common/DnnImageFeaturizer.props#L4-L7

That will copy the model to the output folder of the app.